openapi/docusign.rooms/openapi.yaml

```diff
@@ -0,0 +1,11788 @@
+openapi: 3.0.1
+info:
+  title: DocuSign Rooms API - v2
+  description: DocuSign Rooms streamlines real estate and mortgage workflows by connecting all parties in a digital, secure, and central workspace. With the DocuSign Rooms API, it’s easy to integrate Rooms functionality into your own solutions.
```

Comment:
Shall we follow the standard used in other connectors in the description as follows
https://github.com/ballerina-platform/openapi-connectors/blob/49164bb7baae2ddcd19037c109548143c0fe32d7/openapi/ably/openapi.yaml#L14
https://github.com/ballerina-platform/openapi-connectors/blob/49164bb7baae2ddcd19037c109548143c0fe32d7/openapi/spotify/openapi.yaml#L7